src/GraphQL/DI/IConfigureSchema.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using GraphQL.Types;
+
+namespace GraphQL.DI
+{
+    /// <summary>
+    /// Allows configuration of a schema prior to its constructor.
```

Comment:
> prior to its constructor

I don't understand. What does `prior` mean?

Reply:
Prior = before. 

The code executes before the code in the constructor of MySchema for example.

Reply:
Note: this is because the base class’s constructor is always called before the derived type’s constructor is called. No easy way to work around this. Fortunately most of the time it is desired to run the code first and so set the “default” settings of the schema allowing user code to override it.

Maybe we could add another interface that executes immediately prior to schema initialization, effectively after the constructor. With the current set of extension methods, there was no need.

Reply:
This comment confuses. You mean the descendant constructor. At the same time, the `Schema` constructor code was actually already executed:
```
 _services = services;
Directives = new SchemaDirectives();
Directives.Register(Directives.Include, Directives.Skip, Directives.Deprecated);
```

Better to rephrase the comment.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/2769